benches/run-generator.rs

```
@@ -63,7 +74,7 @@ fn run(c: &mut Criterion) {
                         11000000000,
                         ALLOW_BACKREFS,
                     );
-                    assert!(conds.is_ok());
+                    let _ = black_box(conds);
```

This also ignores the errors.